tests/integration/targets/latest/tasks/ip_address_to_interface.yml

```diff
@@ -116,10 +116,53 @@
     state: absent
   register: test_six
 
-- name: "3 - ASSERT"
+- name: "6 - ASSERT"
   assert:
     that:
       - test_six is changed
       - test_six['diff']['before']['state'] == "present"
       - test_six['diff']['after']['state'] == "absent"
       - "'deleted' in test_six['msg']"
+
+- name: "7 - Add IP address on GigabitEthernet4 - test100 (with ip_address dict)"
+  networktocode.nautobot.ip_address_to_interface:
+    url: "{{ nautobot_url }}"
+    token: "{{ nautobot_token }}"
+    ip_address:
+      address: "10.100.0.1/32"
+      namespace: "Global"
+    interface:
+      name: GigabitEthernet4
+      device: test100
+  register: test_seven
```

Comment:
Please add a duplicate of this task to verify sending `address` and `namespace` are indeed idempotent.

Reply:
Done, thanks. Should be good for review again now.